gradle.properties

```diff
@@ -7,3 +7,19 @@ stdlibConfigVersion=1.0.1-SNAPSHOT
 stdlibIoVersion=0.5.1-SNAPSHOT
 stdlibSystemVersion=0.6.1-SNAPSHOT
 stdlibStringUtilsVersion=0.5.1-SNAPSHOT
+stdlibHttpVersion=1.0.1-SNAPSHOT
+
+stdlibIoVersion=0.5.1-SNAPSHOT
+stdlibStringutilsVersion=0.5.1-SNAPSHOT
+stdlibLogVersion=1.0.1-SNAPSHOT
+stdlibTimeVersion=1.0.2-SNAPSHOT
+stdlibMimeVersion=1.0.1-SNAPSHOT
+stdlibAuthVersion=1.0.1-SNAPSHOT
+stdlibCacheVersion=2.0.1-SNAPSHOT
+stdlibCryptoVersion=1.0.1-SNAPSHOT
+stdlibFileVersion=0.5.2-SNAPSHOT
+stdlibConfigVersion=1.0.1-SNAPSHOT
+stdlibReflectVersion=0.5.1-SNAPSHOT
+stdlibRuntimeVersion=0.5.1-SNAPSHOT
+stdlibSystemVersion=0.6.1-SNAPSHOT
```

Comment:
Shall we remove io (line 12), config (line 21), and system (line 24) from here as well? They are already there

Reply:
sure